content/docs/reference-react-dom-server.md

```diff
@@ -115,8 +111,10 @@ try {
     }
```

Comment:
This could use the `onError` part of the other example too.

Reply:
Oh. I thought try/catch is enough.

Reply:
The try/catch is for onShellError only. onError can fire multiple times (which is why it's an event and not a promise) and still recover by client rendering.

Reply:
`onError` is kind of like `onRecoverableError` on the client. Maybe we should've just called it that.